src/deploy/functions/runtimes/node/parseTriggers.ts

```diff
@@ -377,12 +389,24 @@ export function addResourcesToBuild(
     "timeout",
     proto.secondsFromDuration
   );
+  if (annotation.secrets) {
+    const secretEnvs: backend.SecretEnvVar[] = [];
+    for (const secret of annotation.secrets) {
```

Comment:
nit: Why not:

```
const secretEnvs: backend.SecretEnvVar[] = annotation.secrets.map((secret) => ({
        secret,
        projectId,
        key: secret,
      }));
```

Reply:
ctrl+c/ctrl+v 😛 (see few lines below). I think this one -changed.